### ISSUE TITLE
Gothic 3 (39500)

### ISSUE THREAD

**Issue by lieff** [ValveSoftware/Proton#1257]
Distro: Ubuntu 18.04
GPU: GTX 980, Nvidia proprietary driver 390.48
Resolution: 2560x1440
RAM: 16GB
Proton 3.7-5 Beta

Works without issues.

**Comment by liberodark** [comment]
same for me on 3.16-4 Beta

**Comment by analogueBubblebath** [comment]
Works perfectly
OS: Manjaro 18.0.2 Illyria
Kernel: x86_64 Linux 4.19.16-1-MANJARO
CPU: Intel Xeon E5450 @ 4x 3.003GHz
GPU: GeForce GT 630
RAM: 1900MiB / 7975MiB

**Comment by rokups** [comment]
In my case there is no sound and it horribly stutters when in the game.

OS: Archlinux
Kernel: x86_x64 Linux version 5.1.16-arch1-1-ARCH
CPU: Intel(R) Core(TM) i7-6800K CPU @ 3.40GHz
GPU: RX 580 (amdgpu)
RAM: 32GB
Proton: v4.2

**Comment by kisak-valve** [comment]
Hello @rokups, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

**Comment by rokups** [comment]
In this session stuttering was minimal and game was playable. No sound still.

[steam-39500.log](https://github.com/ValveSoftware/Proton/files/3378306/steam-39500.log)

**Comment by kisak-valve** [comment]
Thanks, can you check if the game has better behavior for you with Proton 3.16-9?

**Comment by rokups** [comment]
[Proton 3.16 (with sound) steam-39500.log](https://github.com/ValveSoftware/Proton/files/3380467/steam-39500.log)
[Proton 4.2 (with sound) steam-39500.log](https://github.com/ValveSoftware/Proton/files/3380488/steam-39500.log)

Sound is OK now. I retried both Proton 3.16 and 4.2 multiple times (switching between versions) and now i have sound. First time i started game i recall i misclicked something in physx installer and it may have not finished. Maybe that was the cause for missing sound.

I also noticed that 3.16 has less stuttering, though there is some. I ran game on 2560x1440 resolution in all tests, other settings of the game unchanged.

**Comment by jarrard** [comment]
Gothic 3 Forsaken Gods Enhanced Edition runs bad with wined3d, and when you enable D9VK it causes most effects/textures/lighting to not render. If you don't delete the intro video files it will crash the desktop most times. 

I've only tried with latest proton version, haven't tested lutris or other proton versions out.

**Comment by jarrard** [comment]
Gothic 3 Forsaken Gods Enhanced Edition appears to work with D9VK now but its performance is pretty bad, must be around 30fps, under windows it seems to be mostly above 60fps. Its a pretty janky game non the less. Not sure if there is a way to improve performance.

**Comment by jarrard** [comment]
Tested this again (FGEE version) and performance is still 20fps or lower with dxvk 1.5 (d9vk is enabled by default). The DXVK_HUD=full command did not work so I couldn't get any other info. Either way this game is having some major issues. (with performance)

Has anyone with a AMD card tried this game? does it run better?

**Comment by jarrard** [comment]
Testing again with DXVK/D9VK 1.5.2 and now I get 5fps.  Renders correctly. 

Going to download Gothic 3 standard edition and see if this is a specific issue with Forsake Gods Enhanced Edition. 

Gothic 3 Quits with runtime error, only the forsaken gods edition works but at 5fps with d9vk. (about 20fps with wined3d)

**Comment by shoober420** [comment]
# Whitelist Request
- Name of the game with compatibility issues: Gothic 3
- Steam AppID of the game: https://steamdb.info/app/39500/

## System Information
- GPU: GTX 760
- Driver/LLVM version: NVIDIA 440.59
- Kernel version: 5.5.2
- https://github.com/shoober420/linux-scripts/blob/main/home/shoober420/sysinfonvidia
- Proton version: 4.11-12

Works great out of the box.

**Comment by shoober420** [comment]
Sorry about the confusion, works great. I had to restart my machine.

**Comment by jarrard** [comment]
Yeah I believe I got it working in the end also. Can't remember how tho.

**Comment by LorenzBung** [comment]
@jarrard If you remember how you fixed it I'd be glad to know.

AMD Vega, 15-25 FPS with `PROTON_USE_D9VK=1`.

**Comment by lieff** [comment]
@LorenzBung Performance issue can be from Gothic3.ttf .
You can google for cleaned version or use another font.

**Comment by LorenzBung** [comment]
@lieff Thanks, that put it to a somewhat playable 30-45 FPS. Still not optimal though.
